stardog-userrole-operator/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,32 @@
+# enabled by default by can be disabled
```

Comment:
The ClusterRoleBinding should only be created if this is enabled in the configuration.